kolibri/plugins/learn/assets/src/views/commonLearnStrings.js

```diff
@@ -4,6 +4,7 @@ export const learnStrings = createTranslator('CommonLearnStrings', {
   // Labels
   learnLabel: 'Learn',
   recommendedLabel: 'Recommended',
+  resumeLabel: 'Resume',
```

Comment:
unfortunately since this is after string freeze, we can't just add a new string like this.

Instead, let's use the `crossComponentTranslator` and pull from `LearnIndex.vue`:

https://github.com/learningequality/kolibri/blob/b751754f79c23990515fa6ebf24d1b3029f57f13/kolibri/core/assets/src/utils/i18n.js#L162-L172

https://github.com/learningequality/kolibri/blob/b751754f79c23990515fa6ebf24d1b3029f57f13/kolibri/plugins/learn/assets/src/views/LearnIndex.vue#L295